app/views/handlers/individual_response.py

```diff
@@ -589,12 +589,17 @@ def handle_post(self) -> Response | None:
 
     def _update_section_completeness(self) -> None:
         if not self._questionnaire_store.progress_store.get_completed_block_ids(
-            section_id=self.individual_section_id, list_item_id=self._list_item_id
+            section_key=SectionKey(
```

Comment:
Non blocker, if this was `section_key := ` instead of being a kwarg, you could reuse it below instead of re-defining it